.github/workflows/ubuntu-lint.yml

```diff
@@ -21,15 +21,14 @@ jobs:
         run: rake setup
       - name: Run Lint
         run: |
-          gem install rubocop -v "~>0.80.1"
-          util/rubocop
+          rake rubocop:rubygems
       - name: Install Dependencies in Bundler
         run: |
           bin/rake spec:deps
         working-directory: ./bundler
       - name: Run Lint in Bundler
         run: |
-          bin/rubocop --ignore_parent_exclusion
+          rake rubocop:bundler
```

Comment:
You should move this to a separate step so that `working-directory: ./bundler` does not apply.